src/components/table/table.tsx

```diff
@@ -71,6 +78,18 @@ export class Table {
     @Event()
     public load: EventEmitter<TableParams>;
 
+    /**
+     * Emitted when a raw is selected
+     */
+    @Event()
+    public selectRow: EventEmitter<number>;
```

Comment:
We should also have an event for when rows have been deselected. In the `limel-list` component all this is handled in the `change` event. If we want to be consistent, we might want to rename this.